src/scripts/install.sh

```diff
@@ -1,5 +1,11 @@
 #!/bin/bash
-curl -o- -L https://slss.io/install | VERSION=$ORB_PARAM_SERVERLESS_VERSION bash
+export export SLS_SCRIPT_URL="https://sls-standalone-sv-1300963013.cos.na-siliconvalley.myqcloud.com/install.sh"
```

Comment:
Extra `export`?

Reply:
Interesting shellcheck didnt yell and it still worked lol. Corrected. Thank you.